library/src/main/java/com/wultra/android/sslpinning/integration/powerauth/PowerAuthCryptoProvider.kt

```diff
@@ -35,8 +36,8 @@ class PowerAuthCryptoProvider : CryptoProvider {
 
     override fun ecdsaValidateSignatures(signedData: SignedData, publicKey: ECPublicKey): Boolean {
```

Comment:
You validate only one signature, so `ecdsaValidateSignature()` would be more appropriate.

Reply:
Renaming this is an API breaking change. But ok, fix for implementations is simple.